form/create_custom_field_type.rst

```diff
@@ -67,14 +67,30 @@ check out the `ChoiceType`_ class.
     extend from ``AbstractType`` is only a convenience way of implementing the
     required ``FormTypeInterface``.
 
-There are three methods that are particularly important:
+There are four methods that are particularly important:
 
 .. _form-type-methods-explanation:
 
+``getParent()``
+    This will defined the type configuring the form before. It means all the
+    other other methods below will be called with this parent type and all its
+    type extensions before calling the ones defined in your custom type.
+    By default, all classes inherit the ``AbstractType`` which defines the
+    ``FormType`` as parent.
+
+``configureOptions()``
+    This defines options for your form type that
+    can be used in ``buildForm()`` and ``buildView()``. There are a lot of
+    options common to all fields (see :doc:`/reference/forms/types/form`),
+    but you can create any others that you need here.
```

Comment:
"[...] but you can create custom options that serve your needs."?

Reply:
What about `but you can create any custom option you need`?

Reply:
sounds good too 👍

Reply:
Done!